packages/eslint-plugin/src/utils/runtime-lint-utils.ts

```diff
@@ -533,6 +564,31 @@ function getAngularEntryPoint(file: string, projectRoot: string): string {
   return undefined;
 }
 
+function getPackageEntryPoints(
+  projectRoot: string
+): Array<{ path: string; file: string }> {
+  const packageContent = readFileIfExisting(
+    path.join(workspaceRoot, projectRoot, 'package.json')
+  );
+  if (!packageContent) {
+    return [];
+  }
+  const exports = parseJson(packageContent).exports;
+  if (!exports) {
+    return [];
+  }
+  const entryPaths: Array<{ path: string; file: string }> = [];
+  for (const [key, value] of Object.entries<string>(exports)) {
+    if (key !== '.' && key !== './') {
+      entryPaths.push({
+        path: joinPathFragments(projectRoot, key),
```

Comment:
The `key` could be the condition name of a [conditional export](https://nodejs.org/docs/latest-v22.x/api/packages.html#conditional-exports).